### PR TITLE
AE-1409 Update NPM package versions and readme md

### PR DESCRIPTION
**Pull request by Nomad-AE**
Updated all of the NPM packages of all of the bots except for the breaking changes of `forta-agent`

Updates NOT implemented:
asset-monitor forta-agent 0.0.39 to 0.1.6
ctoken-monitor forta-agent 0.0.39 to 0.1.6
governance forta-agent 0.0.39 to 0.1.6
large-borrows-governance forta-agent 0.0.39 to 0.1.6
oracle-price-monitor forta-agent 0.0.39 to 0.1.6

### CODE REVIEW

**Review by echoAE — CHANGES_REQUESTED** [review]
Actually, it appears that the newer versions of ESLint are creating additional warnings that need to be addressed before we merged this in.

File: README.md [hunk 1]
```diff
@@ -5,29 +5,63 @@ transactions that occur with Compound Protocol smart contracts.
 
 ## Bots
 
-### Governance Event Monitor
+### cToken Underlying Asset Monitor
```
[the remark this answers is not in the document]
Comment (Nomad-AE) [reply README.md]: Fixed in commit 9a15d242b799fe1838981b426579cf89ca281af6

**Comment by Nomad-AE** [comment]
Added links, linted all js files, and fixed some spelling typos.

**Review by echoAE — APPROVED** [review]

File: multisig-transactions-monitor/package.json [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
   "dependencies": {
-    "forta-agent": "^0.1.1",
-    "bignumber.js": "^9.0.1"
+    "bignumber.js": "^9.0.1",
+    "forta-agent": "^0.1.1"
   },
   "devDependencies": {
     "eslint": "^8.10.0",
-    "eslint-plugin-import": "^2.25.4",
     "eslint-config-airbnb": "^19.0.4",
```
Comment (darian-ae) [inline multisig-transactions-monitor/package.json:29]:
```suggestion
    "eslint-config-airbnb-base": "^15.0.0",
```

Suggesting this for consistency with all the other bots

File: oracle-price-monitor/package.json [hunk 3]
```diff
@@ -22,10 +24,10 @@
     "forta-agent": "^0.0.38"
   },
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint": "^8.16.0",
+    "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.25.1",
```
Comment (darian-ae) [inline oracle-price-monitor/package.json:29]:
```suggestion
    "eslint-plugin-import": "^2.26.0",
```

**Review by darian-ae — APPROVED** [review]
Looks good, the biggest suggestion was changing the `eslint-plugin-import` to `"eslint-plugin-import": "^2.26.0",` for consistency on all the bots

**Pull request merged**